src/components/SearchResults.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/search-results.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p>No results</p>;
+  } else {
+    return (
+      <div className="search-results__image">
+        {results.map((image) => (
+          <img className="card-image" src={image} alt="space" />
```

Comment:
The alt tag needs to be specific to each image. See if there's something in the data you can use. For example- you wouldn't describe each product from amazon as 'TV'